README.md

```diff
@@ -6,5 +6,13 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a back end Web framework and is used as a tool to help us quickly and easily build web applications. Rails assumes what every developer needs to get started on the back end when building a web application. Rails allows you to write less code while accomplishing more than many other languages and frameworks. Rails also allows a web app to communicate with a server.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Routes actually get handled by the server, not the controller, but otherwise this is correct.